addons/binding/pom.xml

```diff
@@ -67,6 +67,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tankerkoenig</module>
```

Comment:
If I understand correctly that should be inserted into the file "addons/binding/pom.xml". The same suggestion has been made by hakan42 and I made this change. Didn't this change make it through git? As far as I can see/understand this pom.xml in the dolic repo has it inserted.